src/nft/pages/collection/index.tsx

```diff
@@ -55,20 +60,27 @@ const Collection = () => {
 
   return (
     <Column width="full">
-      {collectionStats && contractAddress ? (
+      {contractAddress ? (
         <>
           {' '}
           <Box width="full" height="160">
-            <Box
-              as="img"
-              maxHeight="full"
-              width="full"
-              src={collectionStats?.bannerImageUrl}
-              className={`${styles.bannerImage}`}
-            />
+            <Box width="full" height="160">
+              {isLoading ? (
+                <Box height="full" width="full" className={styles.loadingBanner} />
+              ) : (
+                <Box
+                  as="img"
+                  height="full"
+                  width="full"
+                  src={collectionStats?.bannerImageUrl}
+                  className={isLoading ? styles.loadingBanner : styles.bannerImage}
+                  background="none"
+                />
+              )}
+            </Box>
           </Box>
           <Column paddingX="32">
-            {collectionStats && <CollectionStats stats={collectionStats} isMobile={isMobile} />}
+            <CollectionStats stats={collectionStats || ({} as GenieCollection)} isMobile={isMobile} />
```

Comment:
Instead of having to do this typecasted null object, can we have the component `CollectionStats` return null if `collectionStats` is undefined?

Reply:
I had to change this because the loading states exist inside of these components.  I will update the logic so they render if it's loading or collection stats exist so it doesn't render in the case the stats do not exist for some reason